beluga/include/beluga/sensor/likelihood_field_model.hpp

```diff
@@ -161,20 +164,32 @@ class LikelihoodFieldModel {
       return std::min(squared_distance, squared_max_distance);
     };
 
-    const auto to_likelihood = [amplitude =
-                                    params.z_hit / (params.sigma_hit * std::sqrt(2 * Sophus::Constants<double>::pi())),
-                                two_squared_sigma = 2 * params.sigma_hit * params.sigma_hit,
-                                offset = params.z_random / params.max_laser_distance](double squared_distance) {
-      assert(two_squared_sigma > 0.0);
-      assert(amplitude > 0.0);
+    /// Pre-computed variables
+    const double two_squared_sigma = 2 * params.sigma_hit * params.sigma_hit;
+    assert(two_squared_sigma > 0.0);
+
+    const double amplitude = params.z_hit / (params.sigma_hit * std::sqrt(2 * Sophus::Constants<double>::pi()));
+    assert(amplitude > 0.0);
+
+    const double offset = params.z_random / params.max_laser_distance;
+
+    const auto to_likelihood = [=](double squared_distance) {
```

Comment:
I'd suggest capturing only what you need here. It is better to express intent. Copying everything that's in scope is not great (it could potentially include `grid` although inlining probably solves all issues).